test/test_tensor_creation_ops.py

```diff
@@ -1960,6 +1960,88 @@ def test_inference(default_dtype):
         test_inference(torch.float64)
         test_inference(torch.float32)
 
+    @suppress_warnings
+    @onlyCPU
+    def test_tensor_factory_ndarrays(self, device):
+        """
+        Tests torch.tensor for multiple numpy arrays that are embedded (recursively) within list(s). 
+        """
+
+        repeats = 10
+        np_types = list(torch_to_numpy_dtype_dict.values())
+
+        # Simple tests
+        for dtype in np_types:
+            for _ in range(repeats):
+                stack = []
+                stack.append([np.array(1).astype(dtype)])
+                stack.append([np.random.randn(1).astype(dtype)])
+                stack.append([np.random.randn(100).astype(dtype) for _ in range(repeats)])
+                stack.append([np.random.randn(10, 10).astype(dtype) for _ in range(repeats)])
+                stack.append([np.random.randn(100, 1).astype(dtype) for _ in range(repeats)])
+                stack.append([np.random.randn(10, 8, 7).astype(dtype) for _ in range(repeats)])
+
+                for arrays in stack:
+                    self.assertEqual(torch.tensor(arrays, device=device), torch.from_numpy(np.stack(arrays)),
+                                     exact_dtype=True, rtol=1e-05, atol=1e-08)
+
+
+        # Extended tests
+        for dtype in np_types:
+            for _ in range(repeats):
+                stack = []
+                A = np.random.randn(100).astype(dtype)
```

Comment:
Make sure this also tests simple conversions, like a single empty tensor, and tensors with gaps in the memory, like converting tensors where every other element is missing

Reply:
I am not sure what you meant by "converting tensors where every other element is missing". Could you explain further? Like having strides that jump over the odd entries?